app/models/user.rb

```diff
@@ -38,7 +38,7 @@ def matched_id
   end
 
   def partner
-    role == "mentee" ? project.mentee : project.mentor
+    role == "mentee" ? project.mentor : project.mentee
```

Comment:
I need to change it. Since it is a partner function, so it should reverse i.e for mentee it should return mentor. I checked the results of show page and dashboard. Now it was giving correct when i replaced them